pkg/apis/config/defaults.go

```diff
@@ -87,14 +97,20 @@ func NewDefaultsConfigFromConfigMap(config *corev1.ConfigMap) (*Defaults, error)
 }
 
 const (
-	// DefaultRevisionTimeoutSeconds will be set if timeoutSeconds not specified.
+	// DefaultRevisionTimeoutSeconds will be set if RevisionTimeoutSeconds not specified.
```

Comment:
This comment used to refer to the actual spec entry the user sets. Should we leave it as is?